lib/UtapiClient.js

```diff
@@ -861,69 +656,32 @@ export default class UtapiClient {
                     metric: 'storage utilized',
                     error: actionErr,
                 });
-            } else {
-                cmds.push(
-                    ['zremrangebyscore',
-                        genBucketStateKey(bucket, 'storageUtilized', timespan),
-                        timestamp, timestamp],
-                    ['zadd',
-                        genBucketStateKey(bucket, 'storageUtilized', timespan),
-                        timestamp, actionCounter]);
+                return callback(errors.InternalError);
             }
+            cmds.push(
+                ['zremrangebyscore',
+                    genBucketStateKey(bucket, 'storageUtilized'),
+                    timestamp, timestamp],
+                ['zadd', genBucketStateKey(bucket, 'storageUtilized'),
+                    timestamp, actionCounter]);
 
-            // incoming bytes counter
+            // number of objects counter
```

Comment:
incoming bytes has been removed, so I updated the comment. Storage utilized is line 650